regression/cbmc-java/invalid_classpath/test-path.desc

```diff
@@ -0,0 +1,10 @@
+CORE symex-driven-lazy-loading-expected-failure
+Test.class
+--classpath ./NotHere
+^EXIT=6$
+^SIGNAL=0$
+^the program has no entry point$
+^failed to load class `Test'$
+--
+--
+symex-driven lazy loading should emit "the program has no entry point" but currently doesn't
```

Comment:
Probably worth raising a ticket for this

Reply:
@smowton - have you got this covered?

Reply:
done